### PR TITLE
Subscriptions 

### PR DESCRIPTION
**Pull request by julioxavierr**
Error with the new server endpoint /graphiql.

https://github.com/julioxavierr/entria-challenge/blob/30716de8f2a865cbe1f993751adbcd393ff3dc76/src/Environment.js#L12

When I try to `get-graphql-schema` (CLI) it doesn't work either, **it happens using just the boilerplate without modifications too**. Thrown error:

```
{ FetchError: invalid json response body at http://localhost:5000/graphiql reason: Unexpected token < in JSON at position 1
    at /Users/julioxavier/.nvm/versions/node/v9.2.1/lib/node_modules/get-graphql-schema/node_modules/node-fetch/lib/body.js:48:31
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
  name: 'FetchError',
  message: 'invalid json response body at http://localhost:5000/graphiql reason: Unexpected token < in JSON at position 1',
  type: 'invalid-json' }
```

### CODE REVIEW

File: src/scenes/UserList.js [hunk 1]
```diff
@@ -50,13 +78,13 @@ const UserListContainer = createFragmentContainer(UserList, graphql`
...
         query={graphql`
-            query UserListQuery{
+            query UserListQuery {
                 ...UserList_query
             }
         `}
         render={({error, props}) => {
             if (error) {
-                return <Text>Error...</Text>
+                return <Text>{error}</Text>
```
Comment (jgcmarins) [inline src/scenes/UserList.js]: maybe you need to wrap this into `<View>`

File: src/Environment.js [hunk 2]
```diff
@@ -1,12 +1,38 @@
-const {
-    Environment,
-    Network,
-    RecordSource,
-    Store,
-  } = require('relay-runtime');
+import {Environment, Network, RecordSource, Store} from 'relay-runtime';
+import {SubscriptionClient} from 'subscriptions-transport-ws'
+import 'whatwg-fetch'
```
Comment (sibelius) [inline src/Environment.js]: Prefer isomorphic fetch

File: src/scenes/UserList.js [hunk 3]
```diff
@@ -11,9 +11,21 @@ import NewUserButton from '@src/components/NewUserButton';
...
 
+    _subscribe = () => {
+        requestSubscription(
+            environment,
+            {
+                subscription,
+                variables: {},
+                onCompleted: () => console.log('Subscription completed'),
+                updater: (proxyStore) => {
+                    console.log('Subscription updater')
```
Comment (sibelius) [inline src/scenes/UserList.js]: Maybe we need to implemente this

**Review by sibelius — APPROVED** [review]
Great, only missing handling duplicated data

File: src/Environment.js (hunk 2 above)
Comment (jgcmarins) [reply src/Environment.js]: agreed

File: src/components/RowItem.js [hunk 4]
```diff
@@ -12,47 +13,42 @@ export default class RowItem extends Component {
...
-        width: 250,
-        height: 80,
-        fontSize: 30,
-        textAlign: 'left',
-        fontWeight: '800',
-        color: '#508FF2',
-    },
-})
\ No newline at end of file
+Row = styled.TouchableHighlight`
```
Comment (jgcmarins) [inline src/components/RowItem.js]: missing `const` here, right?

File: src/Environment.js (hunk 2 above)
Comment (julioxavierr) [reply src/Environment.js]: Tried, but got:
`Error: Can't find variable: self`
I will go back on this further.

**Review by jgcmarins — APPROVED** [review]
nice
can we merge this?

**Pull request merged**